ormconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "factories": ["src/database/factories/*.ts"]
```

Comment:
This was created with the intention of using factories inside the `__tests__` folder without luck yet. To be continued...